testsuite/basic10-fork-no-constraint/run-tests.tpl

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+{% include 'common-run.sh.tpl' %}
+
+s2e run -n {{ project_name }}
+
+# Make sure this test case generates exactly 4 states.
+# state 0 already exists, state 1~3 are spawned afterwards.
+grep -q "to state 1" $S2E_LAST/debug.txt
+grep -q "to state 2" $S2E_LAST/debug.txt
+grep -q "to state 3" $S2E_LAST/debug.txt
+! grep -q "to state 4" $S2E_LAST/debug.txt
```

Comment:
This is to make sure that there are at most 4 states (state 0~3). State 4 should not exist.

Reply:
It is better to print a message from the guest and grep for that message. This way, the test case will be more stable.

You can use s2e_printf and s2e_get_path_id.